server/src/api/partner.ts

```diff
@@ -43,6 +43,13 @@ export const parseAccountCountry = (
     .with("FRA", "DEU", "ESP", "NLD", "ITA", "BEL", undefined, value => Result.Ok(value))
     .otherwise(country => Result.Error(new UnsupportedAccountCountryError(String(country))));
 
+export const parseAccountCountryStrict = (
```

Comment:
Here I think we can just update the current `parseAccountCountry` function instead of creating a new one because we use this only to parse the query parameter